File: src/main/java/com/haier/uhome/plugins/provider/FlutterTemplateProvider.java
package com.haier.uhome.plugins.provider;

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider;

public class FlutterTemplateProvider implements DefaultLiveTemplatesProvider {

    @Override
    public String[] getDefaultLiveTemplateFiles() {
        return new String[]{"liveTemplates/tools"};
    }

    @Override
    public String[] getHiddenLiveTemplateFiles() {
        return null;
    }
}

File: src/main/kotlin/com/zcy/plugins/provider/FlutterTemplateProvider.kt
package com.zcy.plugins.provider

import com.intellij.codeInsight.template.impl.DefaultLiveTemplatesProvider

class FlutterTemplateProvider : DefaultLiveTemplatesProvider {
    override fun getDefaultLiveTemplateFiles(): Array<String> {
        return arrayOf("liveTemplates/tools")
    }

    override fun getHiddenLiveTemplateFiles(): Array<String?>? {
        return null
    }
}